### DISCUSSION TITLE
Created by dot walk

### DISCUSSION THREAD

**Discussion by attMichal** [arnoudkooi/SN-Utils#293] [Q&A]
Anyway to get the name or manager field from sys_user from a created or updated by dot walk in slushbucket creating a dot walk with show technical names?

**Comment by arnoudkooi** [comment]
Do you mean in a personalized list?
check this video for how to do that https://youtu.be/QM4jOyIiIMk?t=278

**Reply by attMichal** [reply]
Yes exactly! but if you do sys_created_by.manager or sys_updated_by.manager it does not work.

**Reply by arnoudkooi** [reply]
No that's not possible. those are string fields, not reference fields, so you can not dotwalk.
In some tables, you may look for something like opened_by.

**Reply by attMichal** [reply]
Everything is impossible until it becomes possible :)

Initially thought maybe a function to make a ref field but then this gets tedious as I think it would have to be on a lot of tables.
https://docs.servicenow.com/bundle/tokyo-platform-administration/page/build/platform-functions/concept/platform-support-functions.html

Would be a useful enhancement to SN utils if user side we can call gs from personalized list something like:
gs.getUser().getRecord().getValue('manager') or gs.getUser().getRecord().getValue('title') 

Its a real time waster to lookup/sort/view list of who inserted or updated on multiple records

**Reply by arnoudkooi** [reply]
You are aware of selecting the name and then opening the popup? see https://youtu.be/QM4jOyIiIMk?t=110
